required/latex-lab/latex-lab-testphase.dtx

```diff
@@ -83,12 +83,18 @@
 \ProvidesFile{phase-III-latex-lab-testphase.ltx}
               [2023-03-06 v0.1a  latex-lab wrapper phase-III]
 \input{phase-II-latex-lab-testphase.ltx}
-\RequirePackage{latex-lab-testphase-block-tagging}
-\RequirePackage{latex-lab-testphase-sec}
-\RequirePackage{latex-lab-testphase-toc}             
-\RequirePackage{latex-lab-testphase-minipage}             
-\RequirePackage{latex-lab-testphase-graphic}             
-\RequirePackage{latex-lab-testphase-float}             
+\IfFormatAtLeastTF{2023-06-01}
+ {
+  \RequirePackage{latex-lab-testphase-block-tagging}
```

Comment:
thought we wanted to drop -tagging (not important though)

Reply:
yes, next round. I did run out of time  ;-)